docs/deploy/create_server.rst

```diff
@@ -202,6 +202,115 @@ Create the server via the :ref:`host<hosting>`'s interface.
             -  `Windows Server 2019 <https://docs.hetzner.com/robot/dedicated-server/windows-server/windows-server-2019/>`__
             -  `Installing Windows without KVM <https://community.hetzner.com/tutorials/install-windows>`__
 
+   .. tab-item:: Azure
+      :sync: azure
+
+      #. `Log into Azure <https://portal.azure.com>`__
+      #. Click the *Virtual machines* icon
+      #. Click the *Create* menu
+      #. Click the *Azure virtual machine* menu item
+
+         #. Set *Subscription* to "Microsoft Azure Sponsorship (4e98b5b1-1619-44be-a38e-90cdb8e4bc95)"
+         #. Set *Resource group* to the appropriate group (e.g. ``kingfisher``), creating a new resource group, if needed
+         #. Set *Virtual machine name* to the server's FQDN (e.g. ``ocp42.open-contracting.org``)
+         #. Set *Region* to "(Europe) UK South"
+         #. Set *Image* to the latest Ubuntu LTS version
+         #. Set *Size* to the appropriate size (e.g. ``B2s``)
+         #. Set *Authentication type* to "Password"
+         #. Set *Username* to "ocpadmin"
+         #. Set *Password* to a `strong password <https://www.lastpass.com/features/password-generator>`__
+
+      #. Click the *Next : Disks >* button
+
+         #. Set *OS disk type* to *Standard SSD*
+         #. Add additional disks, if appropriate:
+
+            #. Click the *Create and attach a new disk* link
+            #. Click the *Change size* link
+            #. Select "Standard SSD" from the *Storage type* dropdown
+            #. Click the desired size
+            #. Click the *OK* button
+
+      #. Click the *Next : Networking >* button
+
+         #. Set *Virtual network* to an appropriate name with a ``-vnet`` suffix (e.g. ``kingfisher-vnet``)
+         #. Set *Subnet* to *default (10.0.0.0/24)*
+         #. Set *Public IP* to the server's FQDN (e.g. ``ocp42.open-contracting.org``)
```

Comment:
This opens a sidebar with a choice of "Routing preference". Is the default ("Microsoft network") the appropriate option?

<img width="487" alt="Screenshot 2023-11-07 at 4 04 03 PM" src="https://github.com/open-contracting/deploy/assets/26463/4dd44f4a-e22c-4c49-ae3a-f8b24ad3a690">